main/src/main/scala/sbt/Keys.scala

```diff
@@ -325,7 +325,7 @@ object Keys {
   val dependencyClasspathAsJars = taskKey[Classpath]("The classpath consisting of internal and external, managed and unmanaged dependencies, all as JARs.")
   val fullClasspathAsJars = taskKey[Classpath]("The exported classpath, consisting of build products and unmanaged and managed, internal and external dependencies, all as JARs.")
   val internalDependencyConfigurations = settingKey[Seq[(ProjectRef, Set[String])]]("The project configurations that this configuration depends on")
-  private[sbt] val allowZombieClassLoaders = settingKey[Boolean]("Allow a classloader that has previously been closed by `run` or `test` to continue loading classes.")
+  val allowZombieClassLoaders = settingKey[Boolean]("Allow a classloader that has previously been closed by `run` or `test` to continue loading classes.")
```

Comment:
For me the criteria for 1.3.x is more like "is this a bug fix and can I ship it without RC cycle?" In this case, if we're going to ship it I think it should come with this key publicly available.

Reply:
I was under the impression that we promised forward compatibility within the patch release version range. If we don’t, then it makes sense to make this public.

Reply:
We only keep backward compatibility.

Reply:
Ok. I think I misunderstood SemVer. I read the docs on that and see that binary compatible bug fixes are fine under its semantics.